src/alfasim_score/units.py

```diff
@@ -10,11 +11,14 @@
 ROUGHNESS_UNIT = "mm"
 STD_VOLUMETRIC_FLOW_RATE_UNIT = "sm3/d"
 MASS_FLOW_RATE_UNIT = "kg/s"
+MASS_UNIT_SCORE = "lbm"
 PRESSURE_UNIT = "psi"
 TEMPERATURE_UNIT = "degC"
 GAS_OIL_RATIO_UNIT = "sm3/sm3"
 HEAT_TRANSFER_COEFFICIENT_UNIT = "W/m2.K"
 DIMENSIONLESS = "unitless"
 VELOCITY_UNIT = "m/s"
-TIME_UNIT = "min"
+TIME_UNIT = "month"
```

Comment:
Maybe 
```suggestion
TIME_UNIT = "d"
TIME_UNIT_SCORE = "month"
```
Use `TIME_UNIT` for barril Scalars and `TIME_UNIT_SCORE` to get data from SCORE input

Reply:
As only `month` is used, I just changed `TIME_UNIT` to `TIME_UNIT_SCORE`.